src/main/java/lotto/lotto/LottoTicket.java

```diff
@@ -0,0 +1,16 @@
+package lotto.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTicket {
+    List<Lotto> lottoList = new ArrayList<>();
```

Comment:
```suggestion
    List<Lotto> lottos = new ArrayList<>();
```
이미 타입에 list가 표시되어 있으니 변수명에 또 작성할 필요는 없습니다.

Reply:
팁 감사합니다!